colossalai/inference/tensor_parallel/engine.py

```diff
@@ -272,16 +273,16 @@ def generate(self,
             input_tokens = self.tokenizer.batch_encode_plus(prompts, return_tensors="pt", padding=True)
         else:
             input_tokens = prompt_token_ids
-            
+
         if isinstance(input_tokens, list):
-            input_tokens = torch.Tensor(input_tokens).cuda()
+            input_tokens = torch.Tensor(input_tokens).long().cuda()
```

Comment:
What's the purpose of using `long()` here

Reply:
The input data type required for embedding calculation is either int or long.